client/components/tinymce/index.jsx

```diff
@@ -186,11 +188,25 @@ module.exports = React.createClass( {
 
 		this.bindEditorEvents( prevProps );
 
+		if ( this.props.postCategory !== prevProps.postCategory ) {
+			console.log("this.props.postCategory = ", this.props.postCategory);
+			this.updateBodyClass();
+		}
+
 		if ( this.props.mode !== prevProps.mode ) {
 			this.toggleEditor();
 		}
 	},
 
+	updateBodyClass: function() {
+		let postCategory = this.props.postCategory;
```

Comment:
`postCategory` is in the format that's mentioned on https://developer.wordpress.com/docs/api/1.1/get/sites/%24site/posts/%24post_ID/ (see property `categories`)
```
 { 
    "category name 1": categoryObj, 
    "category name 2": categoryObj,
    "category name 3": categoryObj 
    ...
  }
```

However, once you make changes to a Post's Categories **WITHOUT publishing** the Post. `postCategory` is no longer in the format seen above. It is now an array of category IDs.
e.g., 
```
 [ 465154410, 465154522, 380248582, 1591, 7856182 ]
```
I can't seem to find a way to look up Category by its ID... and therefore the rest of the code in this method breaks.